lib/schedule.ex

```diff
@@ -270,6 +269,15 @@ defmodule Schedule do
     {:reply, Data.swings_for_route(gtfs_data, route_id, start_time, end_time), state}
   end
 
+  def handle_call(:peek, _from, {:loaded, gtfs_data} = state) do
+    {:reply, gtfs_data, state}
+  end
+
+  # You probably shouldn't use this in app code, but it is handy for debugging in the REPL
+  def peek(server \\ __MODULE__) do
+    call_catch_timeout(server, :peek, :peek, [])
+  end
+
```

Comment:
Should we remove this before merging? I'd sort of prefer we not leave it if it's only useful in the REPL. Also, you can always do `:sys.get_state([process])` to accomplish basically the same thing.

Reply:
I was thinking of leaving it in, but then, I didn't know about `get_state`, so will remove.